mindsdb/integrations/handlers/file_handler/file_handler.py

```diff
@@ -163,6 +168,19 @@ def _get_data_io(file_path):
         # check for file type
         ############
 
+        # Check first and last 4 bytes equal to PAR1.
+        # Refer: https://parquet.apache.org/docs/file-format/
+        parquet_sig = b'PAR1'
+        data.seek(0, 0)
+        start_meta = data.read(4)
+        data.seek(-4, 2)
+        end_meta = data.read()
+        data.seek(0)
+        if start_meta == parquet_sig and end_meta == parquet_sig:
+            return data, 'parquet', None
+        else:
```

Comment:
Do we need this?

Reply:
Yes, as per parquet's byte signature the first and last 4 bytes should be `PAR1`. Refer here https://parquet.apache.org/docs/file-format/